src/Experiment/tests/BlankAssemblyTest.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Emit.Experimental;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace Experiment.Tests
+{
+    public class BlankAssemblyTest
+    {
+        private readonly AssemblyName _name = new AssemblyName("MyBlankAssembly");
```

Comment:
NIT: instead of declaring a field, you can just pass this as a parameter to your helper function. That way that helper can be made static.